### PR TITLE
L4.9.123 2.3.0 weston

### PR DESCRIPTION
**Pull request by thochstein**

### CODE REVIEW

File: recipes-graphics/wayland/weston/0001-weston.ini-using-argb8888-as-gbm-default-on-mscale-8.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
+From 0ebb3ece5916ec82bbf7c71992244b707eca3f2b Mon Sep 17 00:00:00 2001
+From: Haihua Hu <jared.hu@nxp.com>
+Date: Wed, 4 Jul 2018 16:05:20 +0800
+Subject: [PATCH] weston.ini: using argb8888 as gbm default on mscale 850D
+
+mscale 850D video playback need using pixel alpha for graphic layer
+
+upstream status: imx specific
```
Comment (otavio) [inline recipes-graphics/wayland/weston/0001-weston.ini-using-argb8888-as-gbm-default-on-mscale-8.patch]: Header is wrong.

File: recipes-graphics/wayland/weston/0002-weston.ini-configure-desktop-shell-size-in-weston-co.patch [hunk 2]
```diff
@@ -0,0 +1,30 @@
+From deee5447fea885f9937b55de25f49e5942c5724f Mon Sep 17 00:00:00 2001
+From: Haihua Hu <jared.hu@nxp.com>
+Date: Wed, 11 Jul 2018 17:47:47 +0800
+Subject: [PATCH] weston.ini: configure desktop shell size in weston configure
+ file
+
+mscale 850D need set desktop shell to 1080p
+
+upstream status: imx specific
```
Comment (otavio) [inline recipes-graphics/wayland/weston/0002-weston.ini-configure-desktop-shell-size-in-weston-co.patch]: Wrong header.

**Comment by thochstein** [comment]
Note, I'm having some problems building these changes. Perhaps the changes are dependent on newer graphics drivers.

**Comment by thochstein** [comment]
Okay, I got it working without updating graphics drivers for imx6qsabresd.

**Comment by thochstein** [comment]
Let me know if I can go ahead and squash the marked commits.

**Comment by otavio** [comment]
> Let me know if I can go ahead and squash the marked commits.

Yes @thochstein please go ahead and rework the commits so we can go over them and do the final review.

**Comment by thochstein** [comment]
> > Let me know if I can go ahead and squash the marked commits.
> 
> Yes @thochstein please go ahead and rework the commits so we can go over them and do the final review.

Will let you know when libdrm source is pushed.

**Comment by thochstein** [comment]
libdrm is pushed. This is ready to be merged.

**Comment by otavio** [comment]
Thanks for all the work on this :-)

**Pull request merged**